themes/default/content/docs/concepts/programming-model.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: "Programming Model"
-aliases:
-- /docs/reference/programming-model/
+url: /docs/intro/concepts/programming-model
```

Comment:
We have to add an explicit `url` param here to tell Hugo to render this page at this particular path, since it no longer exists at this location in the `content` tree.